### PR TITLE
OCPBUGS-22899: node spread anti-affinity for HA HCP

### PR DESCRIPTION
**Pull request by sjenning**

### CODE REVIEW

File: support/config/deployment.go [hunk 1]
```diff
@@ -178,15 +178,38 @@ func (c *DeploymentConfig) setMultizoneSpread(labels map[string]string) {
...
+func (c *DeploymentConfig) setAppSpread(app string) {
+	if c.Scheduling.Affinity == nil {
+		c.Scheduling.Affinity = &corev1.Affinity{}
+	}
+	if c.Scheduling.Affinity.PodAntiAffinity == nil {
+		c.Scheduling.Affinity.PodAntiAffinity = &corev1.PodAntiAffinity{}
+	}
+	c.Scheduling.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution = append(c.Scheduling.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution,
+		corev1.WeightedPodAffinityTerm{
+			Weight: 100,
```
Comment (csrwng) [inline support/config/deployment.go:202]: Should it be 150 ?

Comment (sjenning) [reply support/config/deployment.go:203]: Weight is 1-100

**Comment by sjenning** [comment]
This didn't work

**Pull request merged**